config/packages/webpack_encore.yaml

```diff
@@ -0,0 +1,5 @@
+webpack_encore:
```

Comment:
Should it be part of a distribution or do we want everyone to define the defaults, that you've put here? With this config, Sylius (in this repo) will work, but everyone using it in vendor won't access this file (like Sylius-Standard)